tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -216,6 +231,15 @@ function configure_containerd() {
 	done
 }
 
+function restart_service() {
+    if [ -x /sbin/openrc-run ]; then
+        service "$1" restart
+    else
```

Comment:
It might be worth adding a check for systemd here so we can have an `else die "unknown init daemon"` to "handle the impossible" :smile:

Reply:
I am actually giving up on this. I  now understand why you guys stick to systemd for some reason.  Please see this https://github.com/OpenRC/openrc/issues/518. 

Providing the complexity it brings, I think manually installing kata on each host is more feasible way to go. 

Instead of providing a solution, this PR may fallback to document a limit instead. 

For OpenRC fans, if this (https://github.com/kata-containers/kata-containers/issues/4110) could be landed, Go with a dirty hands is probably to the way to go.   I have hacked the Makefile to make kata statically linked with muslc to run it on Alpine Linux.  It built, but not run.  Seem more work to do. Not a simply static link.